client/app/dedicated/server/dashboard/DASHBOARD_VIEW.html

```diff
@@ -110,16 +108,181 @@
+                                    data-text="{{:: 'server_configuration_monitoring_activate_title_button' | translate}}"
+                                    aria-label="{{:: 'server_configuration_monitoring_activate_title_button' | translate}}"
+                                    data-ui-sref="app.dedicated.server.monitoringUpdate"
+                                    data-ng-if="!server.monitored"></oui-action-menu-item>
+                            </oui-action-menu>
+                        </div>
+                    </div>
+                    <div class="oui-tile__item">
+                        <dl class="oui-tile__definition">
+                            <dt class="oui-tile__term"
+                                data-translate="server_configuration_monitoring_services"></dt>
+                            <dd class="oui-tile__description">
+                                <span class="oui-status"
+                                      data-ng-repeat="monitoringProtocol in monitoringProtocolEnum"
+                                      data-ng-class="isMonitoringEnabled(monitoringProtocol) ? 'oui-status_success' : 'oui-status_warning'"
+                                      data-ng-bind="monitoringProtocol"></span>
+                            </dd>
+                        </dl>
+                        <div class="oui-tile__actions"
+                             data-ng-if="!server.isExpired">
+                            <oui-action-menu
+                                aria-label="'menu_action_label' | translate:{t0: ('server_configuration_monitoring_services' | translate)}"
+                                data-align="end"
+                                data-compact>
+                                <oui-action-menu-item
+                                    data-text="{{'server_configuration_monitoring_services_use' | translate }}"
+                                    aria-label="{{'server_configuration_monitoring_services_use' | translate}}"
+                                    data-on-click="currentView.value = 'SERVICES_MONITORING'"></oui-action-menu-item>
+                            </oui-action-menu>
+                        </div>
+                    </div>
+                    <div class="oui-tile__item">
+                        <dl class="oui-tile__definition">
+                            <dt class="oui-tile__term"
+                                data-translate="server_configuration_services_state_links"></dt>
+                            <dd class="oui-tile__description">
+                                <div class="oui-list">
+                                    <ul class="oui-list__items">
+                                        <li class="oui-list__item">
+                                            <a class="oui-list__link oui-button_link"
+                                               href="{{::servicesStateLinks.weathermap}}"
+                                               target="_blank">
```

Comment:
+`rel="noopener"`

Reply:
Some accessibility concerns. Could you please add a `title` attribute which contains: `… ({{ 'core_new_window' | translate }})`?